providers/github/provider/provider.go

```diff
@@ -58,10 +63,13 @@ func (s *Service) ParseCLI(req *plugin.ParseCLIReq) (*plugin.ParseCLIRes, error)
 	// Do custom flag parsing here
 	switch req.Args[0] {
 	case "org":
+		conf.Discover.Targets = []string{connection.DiscoveryOrganization}
```

Comment:
I think, this is not correct. It uses the first arg as discovery target and not what is specified by the discovery flag:

```
cnquery run github org mondoohq --discover repository -c "github.repository.license.spdxId"
```
In this example, `req.Args[0]` is `org`, but I want to discover the repositories.

Reply:
ah gotcha. i didn't realize we actually had discovery targets in v8 for GitHub, it was discovering repos within the org automatically for me without the discovery options -- ill get this all sorted, thanks for testing!